LinkerScript.sublime-syntax

```diff
@@ -5,10 +5,12 @@ file_extensions: [ld]
 scope: source.linker
 
 variables:
-  num_suffix: '[KM]'
+  num_suffix: '[kKM]'
```

Comment:
If we're going to support lower case "k" then I think we should add lower case "m" as well.

Reply:
not really, as uppercase `K` is anyways wrong as SI prefix, whereas uppercase `M` is absolutely correct.